docs/notifications.md

```diff
@@ -0,0 +1,41 @@
+# Notification Development Instructions
+
+The purpose of this document is to provide ZAP developers with simple instructions on how to add a notification.
+
+## Back End
+
+### Schema
+
+Within zap-schema.sql there is a table SESSION_NOTICE which is queried from the front end. SESSION_NOTICE has the following table entries:
+
+![session_notice](schema_notification.png)
+
+**SESSION_REF:** This entry is a key identifier for the current session. It is important because any warning or error that prompts a notification is only relevant to one session.
```

Comment:
This entry is a key identifier for the current session => This entry is a key identifier for a session ?